docs/contributing/contributing-rules.md

```diff
@@ -1,151 +1,415 @@
 
-Resulting input can be similar to the following:
 ```
-pipenv run ./cli/scripts/compare.py 0.103.0 0.106.0 0B1B
+likelihood: LOW
 ```
 
-To use this script with Podman, issue `--use-podman` in the command:
+#### Impact
+
+Indicate how much damage can a vulnerability cause. Use LOW, MEDIUM, and HIGH.
+
+
+##### HIGH
+
+HIGH impact rules can detect extremely damaging vulnerabilities, such as injection vulnerabilities. Examples:
+
+- <LinkToRegistryRule ruleId="javascript.sequelize.security.audit.sequelize-injection-express.express-sequelize-injection" />
+- <LinkToRegistryRule ruleId="ruby.rails.security.audit.xxe.xml-external-entities-enabled.xml-external-entities-enabled" />
 
 ```
-pipenv run ./cli/scripts/compare.py --use-podman 0.103.0 0.106.0 0B1B
-```
\ No newline at end of file
+impact: HIGH
+```
+
+##### MEDIUM
+
+MEDIUM impact rules are issues that are less likely to lead to full system compromise but still are fairly damaging. Examples:
+
+- <LinkToRegistryRule ruleId="python.flask.security.injection.raw-html-concat.raw-html-format" />
+- <LinkToRegistryRule ruleId="python.flask.security.injection.ssrf-requests.ssrf-requests" />
+
+```
+impact: MEDIUM
+```
+
+##### LOW
+
+LOW impact rules are rules which are a security issue, but the impact is not too damaging to the application if discovered. 
+
+- <LinkToRegistryRule ruleId="go.gorilla.security.audit.session-cookie-missing-secure.session-cookie-missing-secure" />
+- <LinkToRegistryRule ruleId="javascript.browser.security.raw-html-join.raw-html-join" />
+
+```
+impact: LOW 
+```
+
+#### Subcategory
+
+Include a subcategory to explain what is the type of the rule. See the subsections below for more details.
+
+##### vuln
+
+A vulnerability rule is something that developers certainly want to resolve. For example, an SQL Injection rule that uses taint mode. Example:
+
+- <LinkToRegistryRule ruleId="javascript.sequelize.security.audit.sequelize-injection-express.express-sequelize-injection" />
+
+```
+subcategory:
+  - vuln
+```
+
+##### audit
+
+An audit rule is useful for code auditors. For example, an SQL rule which finds all uses of the `database.exec(...)` that can be problematic. Example:
+
+- <LinkToRegistryRule ruleId="generic.html-templates.security.unquoted-attribute-var.unquoted-attribute-var" />
+
+```
+subcategory:          
+  - audit
+```
+
+##### guardrail
+
+A guardrail rule, is useful for companies writing custom rules. For example, finding all usages to non-standard XML parsing libraries within the company. The rule can also bring a message that a developer can use only a company-approved library.
+
+```
+subcategory:
+  - guardrail
+```
+
+#### Technology
+
+Any additional information about the technology that gives context to user of the rule. This helps to specify rulesets in Semgrep Registry.
+
+TODO - add rejpresentative rule examples.
```

Comment:
@LewisArdern last TODO, please, what rule you would like to show here as a representative example? 🙂